SDK/AppCenter/Microsoft.AppCenter.iOS.Bindings/ApiDefinition.cs

```diff
@@ -261,6 +264,11 @@ interface MSACAppCenter
         [Static]
         [Export("setCustomProperties:")]
         void SetCustomProperties([NullAllowed] MSACCustomProperties properties);
+
+        // +(void)setMaxStorageSize:(long)sizeInBytes completionHandler(void (^)(BOOL))completionHandler;
```

Comment:
I think we do not need this comment

Reply:
All methods here have Obc-C signature comment

Reply:
We always indicate in the comments the method that we define here. You can see the same comments in other places too.